INZFS.MVC/Services/VirusScan/VirusScanService.cs

```diff
@@ -1,86 +1,58 @@
 ﻿using INZFS.MVC.Models;
 using Microsoft.AspNetCore.Http;
-using nClam;
 using System;
+using System.Diagnostics;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using Cloudmersive.APIClient.NETCore.VirusScan.Api;
+using Cloudmersive.APIClient.NETCore.VirusScan.Client;
+using Cloudmersive.APIClient.NETCore.VirusScan.Model;
+
 
 namespace INZFS.MVC.Services.VirusScan
 {
     public class VirusScanService : IVirusScanService
     {
-        private readonly ClamClient _clam;
-        public VirusScanService(ClamClient clam)
-        {
-            _clam = clam;
-        }
-        public async Task<bool> ScanFile(IFormFile file)
+        private string apikey = Environment.GetEnvironmentVariable("CloudMersiveApiKey");
```

Comment:
Can this be injected please

Reply:
IOptions<Name of the setting class> settingVariable

Reply:
Now added to IOptions under application section